composer.json

```diff
@@ -49,28 +40,51 @@
     "conflict": {
         "drupal/drupal": "*"
     },
-    "extra": {
-        "installer-paths": {
-            "web/core": ["type:drupal-core"],
-            "web/modules/contrib/{$name}": ["type:drupal-module"],
-            "web/profiles/contrib/{$name}": ["type:drupal-profile"],
-            "web/themes/contrib/{$name}": ["type:drupal-theme"],
-            "drush/contrib/{$name}": ["type:drupal-drush"]
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "platform": {
+            "php": "7.3"
         },
+        "preferred-install": {
+            "*": "dist"
+        },
+        "sort-packages": true
+    },
+    "extra": {
         "drupal-scaffold": {
-            "allowed-packages": [
-                "drupal/core"
-            ],
             "locations": {
                 "web-root": "web/"
             },
+            "allowed-packages": [
+                "drupal/core"
+            ],
             "file-mapping": {
                 "[web-root]/.htaccess": {
                     "mode": "replace",
                     "path": "web/core/assets/scaffold/files/htaccess",
                     "overwrite": false
                 }
             }
+        },
+        "installer-paths": {
+            "web/core": ["type:drupal-core"],
+            "web/libraries/{$name}": ["type:drupal-library"],
+            "web/modules/contrib/{$name}": ["type:drupal-module"],
+            "web/profiles/contrib/{$name}": ["type:drupal-profile"],
+            "web/themes/contrib/{$name}": ["type:drupal-theme"],
+            "drush/Commands/contrib/{$name}": ["type:drupal-drush"],
+            "web/modules/custom/{$name}": ["type:drupal-custom-module"],
+            "web/themes/custom/{$name}": ["type:drupal-custom-theme"]
+        },
+        "drupal-core-project-message": {
+            "post-create-project-cmd-message": [
+                "<bg=blue;fg=white>                                                         </>",
+                "<bg=blue;fg=white>  Congratulations, you’ve installed the Drupal codebase  </>",
+                "<bg=blue;fg=white>  from the drupal/recommended-project template!          </>",
```

Comment:
Er, I'm not sure that `drupal/recommended-project` is accurate here?

Also, keep in mind that this composer file is a template that carries through to our projects. Does this type of message make sense for project `composer.json` files?